frontend/doc-site/04__Analyze Public Data/4_4__Annotate.mdx

```diff
@@ -48,47 +45,53 @@ conda activate cellxgene
 ```
 
 ### pip install
-Once your virtual environment is installed and activated, install the `annotate` feature using pip using the following command:
+
+Once your virtual environment is installed and activated, install the `annotate` feature using pip with the following command:
 
 `pip install 'cellxgene[annotate]'`
 
+If `conda` is not installed on your system, install these additional packages:
+`pip install scvi-tools==0.16.2 'scanpy[leiden]==1.9.1' xgboost==1.6.1 torch==1.11.0`
+If this fails, we recommend you use a Conda environment (see the [conda setup](#conda-setup) section, above)
```

Comment:
This is the major change, along with `--mlflow-env-manager local` change, below

Reply:
great - the guidance is good here, and good that redirect people to conda as a last resort :)